packages/keystone/package.json

```diff
@@ -84,6 +86,7 @@
   },
   "repository": "https://github.com/open-condo-software/condo/tree/main/packages/keystone",
   "devDependencies": {
+    "@faker-js/faker": "^7",
```

Comment:
Is that version compatible with the installed one over the apps? Maybe we should sync versions

Reply:
Yep, we have 7th on global package.json